File: level6/1546.c
#include <stdio.h>

int main(){
    int n;

    scanf("%d",&n);

    float arr[n],max=0,sum=0;

    for(int i=0;i<n;i++)
        scanf("%f",&arr[i]);
    max = arr[0];

    for(int i=1;i<n;i++)
        max = max < arr[i] ? arr[i] : max;
    for(int i=0;i<n;i++){
        arr[i] = arr[i]/max*100;
        sum += arr[i];
    }
    printf("%f",sum/n);
    
}

File: baekjoon/js/level4/1546.js
const fs = require('fs')
const filePath = process.platform === 'linux' ? '/dev/stdin' : '../input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

let list = input[1].split(' ').map(Number);
const max = Math.max(...list);
let sum = 0;
list.map((elem) => {
   sum += elem / max * 100 / Number(input[0]); 
})

console.log(sum);

